### ISSUE TITLE
[temperature@fevimu] Applet width jumping on `High` and making the Grouped window list jump

### ISSUE THREAD

**Issue by Fifis** [linuxmint/cinnamon-spices-applets#2538]
```
 * Applet version (Last Modified): 1562780165 (metadata.json)
 * Cinnamon version: 4.2.3
 * Distribution: Arch
 * Graphics hardware: Intel HD Graphics 630
 ```

**Issue**
Unwanted behaviour: after reaching 80°, the taskbar (and all window indicators) starts dancing like crazy because of the ‘High’ text appearing and disappearing after every refresh during the fluctuation.

**Steps to reproduce**
When the temperature gets high or critical (very frequent if projects such as BOINC are running) and then stops around that level.

**Expected behaviour**
Having two new settings: show ‘High’ if an option is set and show ‘Crit’ if an option is set.
A different, slightly more complicated, approach would be moving the text ‘High’ and ‘Crit’ (maybe in colour that the user can pick) below or above the indicator (if such an option is enabled). It would make the applet width fixed, which is a good quality of an applet.

**Comment by leplastic** [comment]
I think this is now solved. There is a second setting "Only use colors to alert me" which hides the text but keeps the background color when reaching the temperature thresholds. This is the setting I'm using and the text is not shown.

**Comment by claudiux** [comment]
@brownsr @Fifis 

This issue was fixed by #2560. Can you close it, please?
